src/compiler/factory/emitHelpers.ts

```diff
@@ -34,6 +34,8 @@ namespace ts {
         // Class Fields Helpers
         createClassPrivateFieldGetHelper(receiver: Expression, privateField: Identifier): Expression;
         createClassPrivateFieldSetHelper(receiver: Expression, privateField: Identifier, value: Expression): Expression;
+        createClassPrivateMethodGetHelper(receiver: Expression, accessCheck: Identifier, fn: Identifier): Expression;
+        createClassPrivateReadonlyHelper(): Expression;

```

Comment:
I'm planning to change the signature of this helper to `createClassPrivateReadonlyHelper(receiver: Expression, privateField: Identifier, value: Expression): Expression;`. This is so that we evaluate the value expression and be more spec-compliant on cases like [this](https://babeljs.io/en/repl#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&spec=false&loose=false&code_lz=MYGwhgzhAEDC0G8BQ1XQMQCMwCcAUAlIgL4poBmA9pYYtAC4AWAlhAHRa7QC80wlAOwiUQAUzYhKAczwAiCI0oBXEABNokqQxYxyzHBHqyCAbmilSSAaIDucQmyo1TQA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=&version=7.12.12&externalPlugins=%40babel%2Fplugin-proposal-private-methods%407.12.1%2C%40babel%2Fplugin-proposal-class-properties%407.12.1). Untranspiled code would print the log first and then throw. Babel transform erases the log.